R/format-table.R

```diff
@@ -11,7 +11,8 @@ source(file.path(getwd(), "R", "constants.R"))
 format_table <-
```

Comment:
Can you leave function descriptions for the `remove_duplicate` and `strip_invalid_part_ID` parameters?